config.json

```diff
@@ -31,11 +31,17 @@
       "defaultValue": false
     },
     "RuntimeOS": {
-      "description": "The RuntimeOS used for building and restoring native OS dependent assets (i.e. RID specific binaries)",
+      "description": "The RuntimeOS of the build system.",
       "valueType": "property",
       "values": ["win7", "osx.10.10", "ubuntu.14.04", "ubuntu.16.04", "etc-other-rid-based-os-names"],
       "defaultValue": ""
     },
+    "PackageRID": {
+      "description": "The RID of the target package.",
```

Comment:
What is the target package in this case? I assume you are implying that it is the RID of what you are building, however I see you a also using it in restoring dependencies.